.github/actions/gradle-and-sha/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Run gradle command and create SHA512
+
+inputs:
+  gradle-command:
+    required: true
+    type: string
+  intermediate-filepath:
+    required: true
+    type: string
+  final-filepath:
+    required: true
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+    - name: "Run gradle command"
+      uses: gradle/gradle-build-action@67421db6bd0bf253fb4bd25b31ebb98943c375e1
+      with:
+        arguments: ${{ inputs.gradle-command }}
+    - name: "Rename zip file"
+      shell: bash
+      run: ls build/jpackage & mv ${{ inputs.intermediate-filepath }} ${{ inputs.final-filepath }}
```

Comment:
Is `jpackage` supposed to be explicitly referenced in this refactored action?

Reply:
No, this was just debugging code since every OS spat out several file options, so this was a way to inspect and see which we wanted (e.g. Windows has MSI and EXE; Mac has DMG and .App). I've removed it now.